scripts/setupLocalIdler.sh

```diff
@@ -55,7 +61,7 @@ EOF
 #   None
 ###############################################################################
 loc() {
-    oc -n dsaas-preview --token ${DSAAS_PREVIEW_TOKEN} $@
+    oc --config $(dirname $0)/config -n dsaas-preview --token ${DSAAS_PREVIEW_TOKEN} $@
```

Comment:
If you use `--config` there should be no need for `--token` and potentially even `-n` could be omitted - if you use it in `oc login` - I think that would create config referencing all - cluster + user + token + namespace

Reply:
right right.

Reply:
fixed